actors/builtin/shared.go

```diff
@@ -91,13 +92,43 @@ type MinerAddrs struct {
 	ControlAddrs []addr.Address
 }
 
+func (b *DeferredCronEventPayload) UnmarshalCBOR(r io.Reader) error {
+	var c bytes.Buffer
+	_, err := c.ReadFrom(r)
+	*b = DeferredCronEventPayload{}
+	return err
+}
+
+func (b DeferredCronEventPayload) MarshalCBOR(w io.Writer) error {
+	_, err := w.Write([]byte("aaaa"))
+	return err
+}
+
+type DeferredCronEventPayload struct {
+	EventType               miner0.CronEventType
```

Comment:
Changes discussed in previous comments will happen here: rename and adding full payload into field.